cmd/gdeploy/commands/sso/sso.go

```diff
@@ -176,12 +179,71 @@ var configureCommand = cli.Command{
 Create a group called 'Granted Administrators' in your identity provider and copy the group's ID.
 Users in this group will be able to manage Access Rules.
 `)
-		adminGroupPrompt := &survey.Input{Message: "The ID of the Granted Administrators group in your identity provider:", Default: dc.Deployment.Parameters.AdministratorGroupID}
-		err = survey.AskOne(adminGroupPrompt, &dc.Deployment.Parameters.AdministratorGroupID)
+
+		// Prompt: Do you want to enter an group id, or select frokm a list of groups?
+		surveyQ := []string{"Select from a list of groups", "Enter group id"}
```

Comment:
Instead of giving an option here, can we always prompt the user to select from a list of groups? I think this could add to confusion for first-time setup as users won't be sure whether they should choose from a list or enter an ID. Additionally, if users want to manually enter an ID they can just edit the `granted-deployment.yml` file by hand.

Reply:
Cheers @chrnorm, think this should be good now. I agree that it makes for a simpler UX (eliminating choice)